### DISCUSSION TITLE
Move from Zenodo to InvenioRDM

### DISCUSSION THREAD

**Discussion by rkrug** [eblondel/zen4R#113] [General]
Hi
I assume, you heard about the move from Zenodo to a next generation platform (https://blog.zenodo.org/2022/12/07/2022-12-07-zenodo-on-inveniordm/).

Just to confirm: zen4R will be adapted to work wit the new API?

Although I am not to clear, if there will be backward compatibility of the API or not:

> **Will your APIs be backward compatibile?**
> 
> Yes, we do our outmost to ensure backward compatibility of our APIs, and your integration will continue to work on the new platform as well.
> 
> **Will I be required to update my API integration ?**
> 
> After Zenodo on InvenioRDM has been launched (Autumn 2023), we will deprecate some of our existing APIs. We will provide a migration period of 1-year for the transition. New features, will only be available on the new API.
> 

Thanks.

**Comment by eblondel** [comment]
To your question "zen4R will be adapted to work wit the new API?"
It will be tested during the transition phase. Zenodo says there would be backward compatibility, but Zenodo team also says some of the APIs methods may be abandonned, well then equivalent methods would be abandonned, and they would need to be replaced or just deprecated.

Practically speaking whether it will be done immediatly will depend on the effort required to do it and opportunities to back this evolutive maintenance. Many parts of zen4R are in-kind contributions to the opensource community, but it doesn't mean it will continue like that. If the zen4R is used, then zen4R dev needs financial support at some point. There will be certainly opportunities to support zen4R to follow-up with the Zenodo to InvenioRDM move.

**Reply by rkrug** [reply]
Thanks a lot for the perspective. zen4R is a brilliant tool and it would really be a shame if it would fade away.

Cheers,

Rainer

**Reply by eblondel** [reply]
This is now actively developed in #127 Most of methods managed in zen4R so far were upgraded, + other new methods added. Soon zen4R will be published on CRAN. You can already use the new zen4R from Github
